src/redux/reducers/app.js

```diff
@@ -76,6 +78,20 @@ export default handleActions(
         userSkills,
       };
     },
+    [actionTypes.APP_LIST_USER_SETTINGS](state, { payload }) {
+      let userName = '';
+      if (payload.settings && payload.settings.userName) {
+        userName = payload.settings.userName;
+      }
+      cookies.set('username', userName, {
```

Comment:
Let's keep reducer for modifying state only, move this to local component

Reply:
@fragm3 Okay.

Reply:
@fragm3 Shall I keep an action associated with it or directly make an api call?